packages/bot-web-ui/src/pages/server-side-bot/add-bot/parts/loss-threshold-warning-dialog.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Checkbox, Dialog } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import { useDBotStore } from 'Stores/useDBotStore';
+import useQsSubmitHandler from '../form-wrappers/useQsSubmitHandler';
+import './loss-threshold-warning-dialog.scss';
+
+const base_classname = 'loss-threshold-warning-dialog';
+
+const LossThresholdWarningDialog = observer(() => {
+    const { quick_strategy } = useDBotStore();
+    const { loss_threshold_warning_data, setLossThresholdWarningData, initializeLossThresholdWarningData } =
+        quick_strategy;
+    const { proceedFormSubmission } = useQsSubmitHandler();
+
+    const handleAmountEdit = () => {
+        setLossThresholdWarningData({
+            show: false,
+            highlight_field: ['loss'],
+        });
+    };
+
+    const handleContinueBot = () => {
+        initializeLossThresholdWarningData();
+        proceedFormSubmission();
+    };
+
+    const handleDontShowAgain = () => {
+        const stored_dont_show_warning_value = localStorage?.getItem('qs-dont-show-loss-threshold-warning');
+        const dont_show_warning = JSON.parse(stored_dont_show_warning_value ?? 'false');
+        localStorage?.setItem('qs-dont-show-loss-threshold-warning', `${!dont_show_warning}`);
```

Comment:
```suggestion
        localStorage.setItem('qs-dont-show-loss-threshold-warning', `${!dont_show_warning}`);
```
in browser env this will never be null or undefined. this is react file so im assuming it is in the browser 😆